harmony-api/src/repos/userRepo.ts

```diff
@@ -1,30 +1,35 @@
 import prisma from '../../prisma/prisma'
 import { User } from '../models/user'
+import { HttpError } from '../models/error/httpError'
 
 export const register = async (userData: User): Promise<User> => {
-  const postResult = await prisma.user.create({
-    data: {
-      username: userData.username,
-      password: userData.password,
-      firstName: userData.firstName,
-      lastName: userData.lastName,
-      active: true,
-    },
-  })
+  try {
+    const postResult = await prisma.user.create({
+      data: {
+        username: userData.username,
+        password: userData.password,
+        firstName: userData.firstName,
+        lastName: userData.lastName,
+        active: true,
+      },
+    })
 
-  // other objects will come from another commit and then I will complete this part
-  return {
-    userId: postResult.userId,
-    username: postResult.username,
-    password: postResult.password,
-    createdAt: postResult.createdAt,
-    active: postResult.active,
-    posts: [],
-    likes: [],
-    comments: [],
-    follows: [],
-    followers: [],
-    firstName: postResult.firstName,
-    lastName: postResult.lastName,
+    // other objects will come from another commit and then I will complete this part
+    return {
+      userId: postResult.userId,
+      username: postResult.username,
+      password: postResult.password,
+      createdAt: postResult.createdAt,
+      active: postResult.active,
+      posts: [],
+      likes: [],
+      comments: [],
+      follows: [],
+      followers: [],
```

Comment:
Not asking for any changes here, but addressing the comment above these lines, I don't think we will need to be returning all of the related objects from the DB, those can be done in separate http requests as needed, that way we can reduce the load on the DB

Reply:
Yeah, I just decided to keep it there until our next discussion